internal/namespaces/instance/v1/custom_runs.go

```diff
@@ -163,3 +170,48 @@ func customInstanceImageListRun(ctx context.Context, argsI interface{}) (i inter
 
 	return customImages, nil
 }
+
+// customInstanceServerTypeListRun transforms the server map into a list to display a
+// table of server types instead of a flat key/value list.
+func customInstanceServerTypeListRun(originalRun core.CommandRunner) core.CommandRunner {
+	return func(ctx context.Context, argsI interface{}) (interface{}, error) {
+
+		type customServerType struct {
+			Name            string        `json:"name"`
+			MonthlyPrice    *scw.Money    `json:"monthly_price"`
+			HourlyPrice     *scw.Money    `json:"hourly_price"`
+			LocalVolumeSize scw.Size      `json:"local_volume_size"`
+			CPU             uint32        `json:"cpu"`
+			GPU             *uint64       `json:"gpu"`
+			RAM             scw.Size      `json:"ram"`
+			Arch            instance.Arch `json:"arch"`
+		}
+
+		originalRes, err := originalRun(ctx, argsI)
+		if err != nil {
+			return nil, err
+		}
+
+		listServersTypesResponse := originalRes.(*instance.ListServersTypesResponse)
+		serverTypes := []*customServerType(nil)
+
+		for name, serverType := range listServersTypesResponse.Servers {
+			serverTypes = append(serverTypes, &customServerType{
+				Name:            name,
+				MonthlyPrice:    scw.NewMoneyFromFloat(float64(serverType.MonthlyPrice), "EUR", 2),
+				HourlyPrice:     scw.NewMoneyFromFloat(float64(serverType.HourlyPrice), "EUR", 3),
+				LocalVolumeSize: serverType.VolumesConstraint.MinSize,
+				CPU:             serverType.Ncpus,
+				GPU:             serverType.Gpu,
+				RAM:             scw.Size(serverType.RAM/1024*1024*1024) * scw.GB, // Manually handle IEC size notation.
```

Comment:
missing parenthesis: `serverType.RAM/1024*1024*1024 != serverType.RAM/(1024*1024*1024)`

nit: use a const `serverType.RAM/GiB`

Reply:
Actually it should be `serverType.RAM*GiB` (or MiB?) since scw.Size is in bytes